cartero-objects/src/request_body_raw.rs

```diff
@@ -196,7 +196,26 @@ mod tests {
     fn test_new() {
         let raw = RequestBodyRaw::new(RequestBodyRawType::Xml, "<?xml?>");
         assert_eq!(raw.payload_type(), RequestBodyRawType::Xml);
-        assert_eq!(raw.payload(), "<?xml?>");
+        assert_eq!(raw.payload().len(), 7);
+        let payload = raw.payload();
+        let contents = String::from_utf8_lossy(payload.as_ref());
+        assert_eq!(contents, "<?xml?>");
+    }
+
+    #[test]
+    fn test_set_payload() {
+        let raw = RequestBodyRaw::new(RequestBodyRawType::Xml, "<?xml?>");
+        assert_eq!(raw.payload_type(), RequestBodyRawType::Xml);
+        assert_eq!(raw.payload().len(), 7);
+        let payload = raw.payload();
+        let contents = String::from_utf8_lossy(payload.as_ref());
+        assert_eq!(contents, "<?xml?>");
+
+        raw.set_payload("<?xml version=\"1.0\"=?>");
+        assert_eq!(raw.payload().len(), 22);
+        let payload = raw.payload();
+        let contents = String::from_utf8_lossy(payload.as_ref());
+        assert_eq!(contents, "<?xml version=\"1.0\"=?>");
     }
```

Comment:
It doesn't make sense to test this anymore, because #233 changed the datatype to String, so this is a noop guaranteed by GObject.